src/lib/arch/aarch64/Instruction.cc

```diff
@@ -178,8 +172,7 @@ const std::vector<uint16_t>& Instruction::getSupportedPorts() {
 }
 
 void Instruction::setExecutionInfo(const ExecutionInfo& info) {
-  if (isInstruction(InsnIdentifier::isLoadMask) ||
-      isInstruction(InsnIdentifier::isStoreAddressMask)) {
+  if (isInsnOneOf(InsnType::isLoad) || isInsnOneOf(InsnType::isStoreAddress)) {
```

Comment:
Shouldn't this be:
`if (isInsnOneOf(InsnType::isLoad, InsnType::isStoreAddress)) {`?

Reply:
Good spot